core/player/src/expressions/evaluator.ts

```diff
@@ -338,12 +560,27 @@ export class ExpressionEvaluator {
         throw new Error(`Unknown expression function: ${expressionName}`);
       }
 
+      if (operator.name === "waitFor" && !options.async) {
```

Comment:
Could we capture `waitFor` as a constant to avoid a miss if it's renamed?

Reply:
The best I think we can do is `operator.name === DEFAULT_EXPRESSION_HANDLERS.waitFor.name`?